lib/pf/profile/filter/all.pm

```diff
@@ -0,0 +1,59 @@
+package pf::profile::filter::profile::filter::all;
```

Comment:
Shouldn't that be pf::profile::filter::all.

If not then I think we'll have a conversation on packages names :P